tests/blackbox/stratisd_cert.py

```diff
@@ -78,7 +78,7 @@ def make_test_filesystem(pool_path, fs_name):
     :return: Object path of the created filesystem
     """
     (
-        (filesystems_created, (array_of_tuples_with_obj_paths_and_names),),
+        (filesystems_created, (array_of_tuples_with_obj_paths_and_names)),
```

Comment:
Please drop this edit from this PR, putting it in a separate PR is fine.

Reply:
That was an automatic formatting thing it looks like. Let me check my version of black.

Reply:
It was a black version issue. I've updated and tested with our current development issue after reverting that and it considers that successfully formatted.